### DISCUSSION TITLE
640 by 480 detect stream ok?

### DISCUSSION THREAD

**Discussion by schford** [blakeblackshear/frigate#9826] [General]
So all my cameras only have a 640 by 480 sub stream.  I just wondered if this is optimal or am I better resizing the main stream to 1280 by 720?

Cheers

Stuart

**Comment by NickM-27** [comment]
it is not optimal but it really depends on the size of the objects expected to be detected. One camera could work just fine on that while another needs the additional resolution

**Reply by schford** [reply]
Thanks Nick, the camera is a fair bit away and up in the air (20 plus feet) so a strange angle. I have put about 1000 images into my model for this camera - would this be affected if I up'd the resolution?

cheers

Stuart

**Reply by NickM-27** [reply]
assuming you mean frigate+, I would not expect any issues if you changed the resolution like you are describing

**Reply by schford** [reply]
yes frigate+ should have been clear. OK I'll give it a shot! Thanks :)
